static/js/components/ObservationPlanRequestLists.jsx

```diff
@@ -109,6 +109,11 @@ const ObservationPlanGlobe = ({ observationplanRequest, loc }) => {
       setObsList(response.data);
     };
     fetchObsList();
+    if (obsList) {
```

Comment:
This just sets the defaults I guess?

Reply:
yes, default for all data points to be unselected.